consumerui/Dockerfile

```diff
@@ -6,11 +6,11 @@ RUN curl -fsSLo /usr/share/keyrings/kubernetes-archive-keyring.gpg https://packa
 
 RUN wget https://github.com/cloud-ark/kubeplus/raw/master/kubeplus-kubectl-plugins.tar.gz && gunzip kubeplus-kubectl-plugins.tar.gz && tar -xvf kubeplus-kubectl-plugins.tar && cp -r /plugins/* bin/
 
-ADD requirements.txt /root/requirements.txt
-ADD consumerui.py /root/consumerui.py
-ADD grapher.py /root/grapher.py
-ADD templates /root/templates
-ADD static /root/static
+ADD consumerui/requirements.txt /root/requirements.txt
```

Comment:
Why is this change needed?

I see this type of change in all the Dockerfiles below. Why is that needed? 

Won't this affect local builds which are needed during development?

Reply:
We are building the binaries within the Dockerfile, and the context is at the base of the repository.  We could have kept the context within the service folders but  I think some of the go applications reference components in the base of the project and Docker does not allow us to do something like `../`

Reply:
What a believe we need to do is update the scripts used to build the development images and have the context set to the base of the project.

Reply:
I went through the matrix strategy docs. I am still unclear on how the local builds will happen.

The current workflow to test any new code change is as follows:
- make code changes
- use the build scripts to create the latest tag
- change the appropriate container in the kubeplus deployment yaml in deploy/kubeplus-chart/template to use the latest tag
- deploy the kubeplus chart from deploy/kubeplus-chart

The changes to the Dockerfiles will break this workflow.